Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM debian:bookworm-slim
+WORKDIR /var/www/i.upmath.me
+
+RUN apt-get update && apt-get -y install \
+    texlive-full \
```

Comment:
Actually, I've installed vanilla TeX Live, like explained here: https://tex.stackexchange.com/questions/1092/how-to-install-vanilla-texlive-on-debian-or-ubuntu

I did this from the beginning because the packages had too old versions of the required programs, especially dvisvgm.

There are several caveats associated with this installation method. For example, TeX Live creates font-related files in the user's local folders. One has to make sure that the folders have the appropriate permissions. I think the texlive-full package may have its own subtleties that I don't know about right now.

Anyway, let's leave it as is for now. Doing a vanilla TeX Live installation in the docker is tricky.